tools/nats-echo/nats-echo.go

```diff
@@ -65,11 +66,24 @@ func main() {
 	opts := []nats.Option{nats.Name("NATS Echo Service")}
 	opts = setupConnOptions(opts)
 
+	if *userCreds != "" && *nkeyFile != "" {
+		log.Fatal("specify -seed or -creds")
+	}
+
 	// Use UserCredentials
 	if *userCreds != "" {
 		opts = append(opts, nats.UserCredentials(*userCreds))
 	}
 
+	// Use Nkey authentication.
+	if *nkeyFile != "" {
+		opt, err := nats.NkeyOptionFromSeed(*nkeyFile)
+		if err != nil {
```

Comment:
again ignore err in all examples and let Connect pick it up?

Reply:
same comment - I think the examples should show error handling, if what we think we are saying is that adding an option shouldn't return an error until a connect, we need to address that instead

Reply:
+derekcollison Added goreleaser so the release page has pre-built binaries like we do for nsc, ngs, etc. The current release binaries date back to june 6, 2017!

Reply:
I believe Connect will return the error, so you don't have to process in place and can make the example a little shorter. Be good to try out though..

Reply:
Without the error, you get the error: `nats: Authorization Violation`
With the error checking you get an error that is much better `nats: open /tmp/foo: no such file or directory`

Reply:
Since this is tooling and not a code snippet I think the targetted error message is much better.

Reply:
ok, wonder if we should return error for creds option as well.